tests/wallet/test_coin_selection.py

```diff
@@ -80,6 +80,69 @@ async def test_coin_selection_randomly(self, a_hash: bytes32) -> None:
             )
             assert result is not None
             assert sum([coin.amount for coin in result]) >= target_amount
+            assert len(result) <= 500
+
+    @pytest.mark.asyncio
+    async def test_coin_selection_with_dust(self, a_hash: bytes32) -> None:
```

Comment:
Can we also add one more case when we have a bunch of smaller coins let's say 2000 mojos, but no huge coin?

Reply:
That test is below